vehicle/raw_vehicle_cmd_converter/src/accel_map.cpp

```diff
@@ -43,43 +43,43 @@ bool AccelMap::readAccelMapFromCSV(const std::string & csv_path)
 
 bool AccelMap::getThrottle(const double acc, double vel, double & throttle) const
 {
-  std::vector<double> accs_interpolated;
+  std::vector<double> accelerations_interpolated;
```

Comment:
acceleration is not coutable.
How about interpolated_acc_vec or so.

Reply:
@takayuki5168
yes but it's still acceleration unit interpolated and not velocity unit.